cdap-app-fabric/pom.xml

```diff
@@ -253,6 +253,11 @@
       <artifactId>spark-core_2.11</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-features</artifactId>
```

Comment:
Do you still need this?

Reply:
What are we using in AbstractContext that belongs to the cdap-features module?

Reply:
We are using a Feature Flag to guard adding the "feature." configuration options into runtime arguments.